app/models/host/managed.rb

```diff
@@ -112,6 +112,7 @@ class Jail < ::Safemode::Jail
   # audit the changes to this model
   audited :except => [:last_report, :puppet_status, :last_compile], :allow_mass_assignment => true
   has_associated_audits
+  self.auditing_enabled = !Foreman.in_rake?('db:migrate')
```

Comment:
`audited` method call unfortunately always adds this [class_attribute](https://github.com/collectiveidea/audited/blob/v3.0.0/lib/audited/auditor.rb#L51) and sets it to [true](https://github.com/collectiveidea/audited/blob/v3.0.0/lib/audited/auditor.rb#L90) so it would override the value from ActiveRecord::Base